src/configs.rs

```diff
@@ -47,3 +62,41 @@ pub const FIXED_SEED: [u8; 32] = [
     89, 15, 245, 34, 234, 120, 17, 218, 167, 20, 216, 9, 59, 62, 123, 217, 29, 137, 138, 115, 62,
     152, 136, 135, 48, 127, 151, 205, 40, 7, 51, 131,
 ];
+
+pub fn on_string_config_set(
+    config_ctx: &ConfigurationContext,
+    name: &str,
+    val: &'static ValkeyGILGuard<ValkeyString>,
+) {
+    let v = val.get(config_ctx);
+    let value_str = v.to_string_lossy();
+
+    let value = match value_str.parse::<f64>() {
+        Ok(v) => v,
+        Err(_) => {
+            let _: Result<(), ValkeyError> = Err(ValkeyError::Str("Invalid floating-point value"));
+            return;
+        }
+    };
+
+    match name {
+        "bloom-fp-rate" => {
+            if !(BLOOM_FP_RATE_MIN..=BLOOM_FP_RATE_MAX).contains(&value) {
+                let _: Result<(), ValkeyError> = Err(ValkeyError::Str(utils::BAD_ERROR_RATE));
+                return;
+            }
+            BLOOM_FP_RATE.set(config_ctx, v.clone());
+        }
+        "bloom-tightening-ratio" => {
+            if !(TIGHTENING_RATIO_MIN..=TIGHTENING_RATIO_MAX).contains(&value) {
+                let _: Result<(), ValkeyError> = Err(ValkeyError::Str(utils::BAD_ERROR_RATIO));
+                return;
+            }
+            TIGHTENING_RATIO_CONFIG.set(config_ctx, v.clone());
```

Comment:
We need to change the f64 config here...
```suggestion
        let mut tightening = BLOOM_TIGHTENING_RATIO.lock().expect("We expect the tightening_ratio static to exist.");
        *tightening = value;
```